wfl/Models/StatusSearchCriteria.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using MongoDB.Driver;
+using System.Threading.Tasks;
+
+namespace Symphono.Wfl.Models
+{
+    public class StatusSearchCriteria: ICriteria<FoodOrder>
+    {
+        public FoodOrder.StatusOptions Status { get; set;}
+
+        public async Task<IEnumerable<FoodOrder>> ApplyCriteria(IMongoCollection<FoodOrder> collection)
+        {
+            IAsyncCursor<FoodOrder> task = await collection.FindAsync(o => o.Status == this.Status);
+            IList<FoodOrder> orders = task.ToList();
```

Comment:
@JChapman202 One other option would be to return a MongoDB filter directly from this method, which can be passed into the call to FindAsync from the db manager. That would also help get rid of copy/paste.